examples/borer-circe-migration/src/main/scala/btest/TranscodingCreateOptional.scala

```diff
@@ -0,0 +1,36 @@
+package pl.marconzet
```

Comment:
Lemme proceed with further sources once the existing comments are resolved since I guess some stuff is gonna change around here...